x-pack/test/api_integration/apis/maps/migrations.js

```diff
@@ -51,11 +51,15 @@ export default function ({ getService }) {
 
     describe('embeddable migrations', () => {
       before(async () => {
-        await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/maps/kibana');
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/maps/data.json'
```

Comment:
Mind if we rename this to `kbn_archiver/maps.json`. We don't need a directory like there was with esArchiver since it's just one file.